airbyte-integrations/connectors/source-klaviyo/source_klaviyo/schemas.py

```diff
@@ -0,0 +1,104 @@
+#
+# MIT License
+#
+# Copyright (c) 2020 Airbyte
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+
+from datetime import datetime
+from typing import Any, List, MutableMapping, Optional
+
+from pydantic import BaseModel, Extra
+
+
+class BaseSchemaModel(BaseModel):
+    class Config:
+        extra = Extra.allow
+
+        @staticmethod
+        def schema_extra(schema: MutableMapping[str, Any], model) -> None:
+            for prop in schema.get("properties", {}).values():
+                prop.pop("title", None)
```

Comment:
why are we popping the title from schemas? Can you leave a comment?

Reply:
Pydantic by default add a title to every attribute, agree, I can add the comment explaining this